src/components/presentations/Signup/__test__/Signup.test.jsx

```diff
@@ -9,7 +9,9 @@ describe('Login component', () => {
   });
 
   it('always renders a div', () => {
-    const wrapper = shallow(<Signup auth={{ isLoading: false }} />);
+    const wrapper = shallow(
+      <Signup isLoadingReducer={{ loader: true }} user={{ userProfile: {} }} />
```

Comment:
We can create a `props` object and spread it into this component in the test

Reply:
Same as ☝️ 

You haven't resolved this yet. Why was this marked as resolved? 🤔